lib/kitchen.rb

```diff
@@ -70,16 +70,27 @@ def source_root
     #
     # @return [Logger] a logger
     def default_logger
-      Logger.new(:stdout => $stdout, :level => env_log)
+      Logger.new(:stdout => $stdout, :level => Util.to_logger_level(env_log))
```

Comment:
I actually thought it would be cleaner to move all this logging config from `lib/kitchen.rb` to `lib/kitchen/logger.rb` - thoughts?